libs/sdk-ui-dashboard/src/presentation/layout/DashboardLayoutWidget.tsx

```diff
@@ -128,12 +128,17 @@ export const DashboardLayoutWidget: IDashboardLayoutWidgetRenderer<
 
     const handleDragEnd = useWidgetDragEndHandler();
 
+    // TODO: we should probably do something more meaningful when item has no widget; should that even
+    //  be allowed? undefined widget will make things explode down the line away so..
+    const widget = item.widget()!;
+    const isDraggableWidgetType = !(isPlaceholderWidget(widget) || isCustomWidget(widget));
```

Comment:
could we move this method to "../../widgets"; where other widgets check function are placed? Maybe rename it to isDraggableWidget?